### ISSUE TITLE
Faire une vidéo de présentation

### ISSUE THREAD

**Issue by Minishlink** [Minishlink/physbook#69]
- [ ] Version teaser (40 secs max)
- [ ] Version normale (2:30 max)

**Comment by batgau71** [comment]
Voir la dedans pour des modèles : http://shareae.com/after-effects-project/infographics/

**Comment by astridfrot** [comment]
Simplicité, mots clés, logo, slogan, style épuré et infographics

**Comment by Hcops4** [comment]
ca fait un an?

**Comment by Minishlink** [comment]
Yep, sur le Drive j'avais commencé une vidéo longue qui explique les fonctionnalités. Version brouillon sur Youtube (compte Gadz) : https://www.youtube.com/watch?v=_i1bSQVAjAA
